src/main/windows/callsWidgetWindow.ts

```diff
@@ -218,5 +216,28 @@ export default class CallsWidgetWindow extends EventEmitter {
 
         this.setBounds(initialBounds);
     }
+
+    private onPopOutOpen = () => {
+        return {
+            action: 'allow' as const,
+            overrideBrowserWindowOptions: {
+                autoHideMenuBar: true,
+            },
+        };
+    }
+
+    private onPopOutCreate = (win: BrowserWindow) => {
+        this.popOut = win;
+    }
+
+    private onPopOutFocus = () => {
+        if (!this.popOut) {
+            return;
+        }
+        if (this.popOut.isMinimized()) {
+            this.popOut.restore();
+        }
+        this.popOut.focus();
```

Comment:
@tboulis To test this you'd likely need to do it locally since this requires a new version of the plugin compiled with the changes in the PR I linked above since we are sending a new event.